tests/migration_test.go

```diff
@@ -658,7 +658,7 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 			// Previously, we'd stop getting events after libvirt reconnect, which
 			// prevented things like migration. This test verifies we can migrate after
 			// resetting libvirt
-			It("[test_id:4746][QUARANTINE]should migrate even if libvirt has restarted at some point.", func() {
```

Comment:
We should remove the `[QUARANTINE]` tag after we have checked that the fix worked and the test is actually stable, you can read about the quarantine procedure here https://github.com/kubevirt/kubevirt/blob/main/docs/quarantine.md

Reply:
Hey Federico, 
Thanks for the note, As soon as this PR passes presubmit lanes I will return quarantine and stop holding this pr.

Reply:
awesome thanks!